src/client/containers/MessageInput.jsx

```diff
@@ -10,7 +10,8 @@ class MessageInput extends React.Component {
     const user = this.props.app.username;
 
     if (event.keyCode === 13 && text) {
-      this.props.dispatch(sendMessage({ text }));
+      let timeStamp = moment().fromNow();
```

Comment:
Please remove, this content has been added to the refactored MessageInput component